ssg/build_yaml.py

```diff
@@ -570,6 +570,48 @@ def from_yaml(yaml_file, env_yaml=None):
         rule.validate_references(yaml_file)
         return rule
 
+    def add_product_specific_items(self, product):
```

Comment:
I think this method should have a different name, because it doesn't add anything, it does the opposite - it removes items that are not relevant to the product.